test/runner.py

```diff
@@ -494,6 +521,7 @@ def run_graph_tests(meta, allocator, endpoints, tests):
         ['--allocator', allocator],
         default_ipc_socket=VINEYARD_CI_IPC_SOCKET,
     ) as (_, rpc_socket_port):
+        run_graph_extend_test(tests)
```

Comment:
Put this after arrow fragment test, since the test order should from simple to complex